deltachat-ios/MessageKit/Views/Cells/MediaMessageCell.swift

```diff
@@ -73,12 +81,30 @@ open class MediaMessageCell: MessageContentCell {
             imageView.image = mediaItem.image ?? mediaItem.placeholderImage
             playButtonView.isHidden = true
         case .video(let mediaItem):
-            imageView.image = mediaItem.image ?? mediaItem.placeholderImage
+            if let url = mediaItem.url {
+                if let image = mediaItem.image {
+                    imageView.image = image
+                } else {
+                    // no image in cache
```

Comment:
no placeholder is set in this case, it's just the cells background and the play button.

Reply:
> no placeholder is set in this case, it's just the cells background and the play button.

Do you mean in the `else`-case? In .loadVideoThumbnail I give an optional placeholder as argument and set the placeholder there. Not sure what I should change here?

Reply:
oh, you're right. I've overseen that. Thanks for the hint